edge-apps/edge-apps-library/src/utils/oauth.ts

```diff
@@ -0,0 +1,19 @@
+/**
+ * Retrieves an OAuth access token from the Screenly OAuth service
+ * @returns The access token for the configured OAuth provider
+ */
+export const getAccessToken = async (): Promise<string> => {
+  const response = await fetch(
+    screenly.settings.screenly_oauth_tokens_url + 'access_token/',
```

Comment:
```suggestion
export const getAccessToken = async (): Promise<string> => {
  const response = await fetch(
    screenly.settings.screenly_oauth_tokens_url + 'access_token/',
```

It's not always `access_token`, it depends on the name of the `setting` itself. So I think better to make it as an argument and `access_token` as a default value.